public/app/plugins/datasource/loki/components/LokiContextUi.tsx

```diff
@@ -253,8 +252,8 @@ export function LokiContextUi(props: LokiContextUiProps) {
     };
   }, []);
 
-  // Currently we support adding of parser and showing parsed labels only if there is 1 parser
-  const showParsedLabels = origQuery && isQueryWithParser(origQuery.expr).parserCount === 1 && parsedLabels.length > 0;
+  // If there's any "parsedLabel", that also includes structured metadata, we show the parsed labels input
```

Comment:
```suggestion
  // If there's any nonIndexed labels, that includes structured metadata and parsed labels, we show the nonIndexed labels input
```

Reply:
Good catches! I renamed everything in c84e359e0ac